libraries/botframework-streaming/tests/NamedPipe.test.js

```diff
@@ -258,20 +254,44 @@ describe.windowsOnly('Streaming Extensions NamedPipe Library Tests', function ()
             expect(() => client.disconnect()).to.not.throw();
         });
 
-        // TODO: 2023-04-24 [hawo] #4462 The client.send() call will only resolve when the other side responded.
-        //                         Because the other side is not connected to anything, thus, no response is received.
-        //                         Thus, the Promise is not resolved.
-        it.skip('sends without throwing', function (done) {
-            const req = new StreamingRequest();
-            req.Verb = 'POST';
-            req.Path = 'some/path';
-            req.setBody('Hello World!');
-            client
-                .send(req)
-                .catch((err) => {
-                    expect(err).to.be.undefined;
-                })
-                .then(done);
+        it('sends without throwing', function (done) {
+            const response = new StreamingResponse();
+            response.statusCode = 111;
+            response.setBody('Test body.');
+            class TestRequestHandler extends RequestHandler {
+                constructor() {
+                    super();
+                }
+                processRequest(_request, _logger) {
+                    return response;
+                }
+            }
+            const server = new NamedPipeServer('pipeClientTest', new TestRequestHandler());
+            const client = new NamedPipeClient('pipeClientTest', false);

```

Comment:
here the false is wrong, because the second parameter is the handler.

Reply:
Done. Thanks!